JavaFXUserInterface/src/main/resources/AddAsset.fxml

```diff
@@ -107,6 +107,8 @@
                     <Font name="Segoe UI" size="14.0"/>
                 </font>
             </Text>
+         <Button fx:id="uploadBtn" layoutX="650.0" layoutY="97.0" mnemonicParsing="false" text="Upload image" styleClass="uploadBtn"/>
```

Comment:
The upload button seems to be overlapping with the input validation messages. It would be a good idea to move it farther up to the side. 
![inputvalidation](https://user-images.githubusercontent.com/47158909/113590950-4dd6f980-9601-11eb-9d53-afda399382a5.PNG)